package.json

```diff
@@ -53,9 +54,15 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint && yarn test"
+      "pre-commit": "lint-staged",
+      "post-commit": "git update-index --again",
+      "pre-push": "yarn test"
```

Comment:
@olslash thoughts on only running tests on push?  They can take a while and really eat up commit time.

Reply:
this one I'm not ok with; we want to prevent committing broken code in the first place, this doesn't help that. Just encourages committing whatever and then making a junk "fixes" commit before pushing.

Reply:
Or making a junk fixes commit and then rebasing the whole thing to make a lot more sense

.... ok so very few will ever do that.

It's just that commits take SOOO long right now.  Maybe it's just me cuz most of my commits are on code review / refactoring time.  And it sucks to be stuck there for 2 minutes for each commit when I'm going to be doing a whole bunch of merges and stuff.

Reply:
Unfortunately we can't do the equivalent of lint-staged for tests.  No way to zero in on "just run tests that MIGHT have broken in this commit."

Sigh.... Ok I'll put tests back to pre-commit then.  I guess pre-push is quite redundant cuz they're gonna get run by jenkins post-push anyway

Reply:
the time savings from only running eslint on staged files are pretty big, so we're net gain anyway